.github/workflows/security-scan.yaml

```
@@ -0,0 +1,17 @@
+name: Security scan
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+      - hotfix/*
+      - work/secscan  # For development
+
+jobs:
+  python-scans:
+    name: Scan Python project
+    uses: lengau/starflow/.github/workflows/scan-python.yaml@work/secscan
```

TODO after merging https://github.com/canonical/starflow/pull/3:
```suggestion
    uses: canonical/starflow/.github/workflows/scan-python.yaml@main
```